packages/xo-web/src/xo-app/vm-import/esxi/vm-data.js

```diff
@@ -0,0 +1,57 @@
+import _ from 'intl'
+import React from 'react'
+import { Col, Row } from 'grid'
+import { formatSize } from 'utils'
+
+const VmData = ({ data }) => (
+  <div>
+    <Row>
+      <Col mediumSize={6}>
+        <div className='form-group'>
+          <label>{_('vmNameLabel')}:</label> <span>{data.nameLabel}</span>
```

Comment:
The `:` should be part of the i18n because the space before it changes depending on the language. We have a `keyValue` message exactly for that case.